patches/tModLoader/Terraria/Player.TML.cs

```diff
@@ -11,6 +11,9 @@ public partial class Player {
 
 		public HashSet<int> nearbyModTorch = new HashSet<int>();
 
+		internal ModPlayer.Regen modManaRegen = ModPlayer.Regen.Create();
+		internal ModPlayer.Regen modLifeRegen = ModPlayer.Regen.Create();
```

Comment:
Do these actually need to be fields? They're only used in the one method, and they're `internal` so they should be local variables instead.

Reply:
No, no they don't, I need to watch for that better instead of fielding things all the time to start. Fixed.